crates/mxd-verification/Dockerfile

```
@@ -0,0 +1,19 @@
+# Dockerfile for running the TLC model checker
+# Uses TLA+ tools pinned at v1.7.4 for reproducibility
+
+FROM docker.io/eclipse-temurin:21-jre-alpine
+
+# Create non-root user for security
+RUN adduser -D -u 1000 tlc
+
+# Download tla2tools.jar pinned at v1.7.4 (update version here when upgrading)
+RUN apk add --no-cache curl && \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)